client/src/pages/Home.tsx

```diff
@@ -87,6 +107,47 @@ function Home(this: any) {
     }
   };
 
+  const setEmotionStyle = () => {
+    switch (emotionLabel) {
+      case "Happy":
+        return {
+          backgroundColor: "lightgreen",
```

Comment:
The background colour is a bit jarring to look at, will probably update with a coloured border or a button-like design.

Reply:
We can change that later